tests/session/cmd/test_show_config.py

```diff
@@ -262,7 +264,7 @@ def test_show_config_matching_env_section(tox_project: ToxProjectCreator) -> Non
 
 
 def test_package_env_inherits_from_pkgenv(tox_project: ToxProjectCreator, demo_pkg_inline: Path) -> None:
-    project = tox_project({"tox.ini": "[pkgenv]\npass_env = A, B\ndeps=C\n D"})
+    project = tox_project({"tox.ini": "[pkgenv]\npass_env = A, AA\ndeps=C\n D"})
```

Comment:
Why do you change this test?

Reply:
I did include a comment about it in my commit:

Because AppData is now included on Windows, it will include A, AppData, B when testing on Windows
therefore by using A, AA instead it will have the right flow/order.

Reply:
It may be that my comment is not sufficient enough. I will try to explain it a bit better.
Because AppData is included in the passed environment variables per default on Windows since the pass_env is [A, B, ...] in the test, AppData will be included between A and B causing the test to fail: [A, AppData, B, ...]
So therefore changing it to [A, AA] solved it. It was not the most beautiful solution so It could be considered if I should try to find a different value like [K, L] if that is preferred? 
I was considering using [1, A] but that also looked messy. 

Does that make sense?